src/DIRAC/Interfaces/API/Job.py

```diff
@@ -487,6 +486,30 @@ def setCPUTime(self, timeInSecs):
         self._addParameter(self.workflow, "CPUTime", "JDL", timeInSecs, description)
         return S_OK()
 
+    #############################################################################
+    def setPriority(self, priority: int):
```

Comment:
After `from DIRAC.Core.Utilities.ReturnValues import DReturnType`:

```suggestion
    def setPriority(self, priority: int) -> DReturnType:
```